packages/gui/src/components/Navigation.tsx

```diff
@@ -260,8 +260,9 @@ const Navigation = () => {
               wrap="nowrap"
             >
               {window.activeWorkspace && (<Grid item>
-                <Tooltip title={`Toggle Global vs Workspace Scope (${globalScope ? 'Global' : 'Workspace'} Scope)`}>
+                <Tooltip title="Toggle Global vs Workspace Scope (Workspace Scope)">
```

Comment:
Can we revert this change? The tooltip should show which scope is currently activated.

Reply:
Oh sorry! This was me trying to figure out why the test broke. Turned out it was unrelated, but I forgot to undo it.